examples/requestreply/main.go

```diff
@@ -17,7 +17,7 @@ import (
 	"time"
 
 	"github.com/lytics/grid"
-	etcdv3 "github.com/coreos/etcd/clientv3"
```

Comment:
We'll have to do this for ALL our dependencies if we do this otherwise DEP pulls in two copies of the dependency.

Reply:
Semi-related: Go modules replaces `dep`, right?

Reply:
@mattayes Yep.

Reply:
@mattayes Should we update our `dep`-related documentation (see testing section of README) as well?

Reply:
@mattayes Good call - taking a look